superset-frontend/cypress-base/cypress.json

```diff
@@ -1,13 +1,14 @@
 {
   "baseUrl": "http://localhost:8081",
   "chromeWebSecurity": false,
-  "defaultCommandTimeout": 20000,
-  "requestTimeout": 20000,
-  "ignoreTestFiles": ["**/!(*.test.js)"],
-  "projectId": "fbf96q",
+  "defaultCommandTimeout": 5000,
+  "requestTimeout": 10000,
+  "ignoreTestFiles": [
+    "**/!(*.test.js)"
+  ],
   "video": false,
   "videoUploadOnPasses": false,
   "viewportWidth": 1280,
   "viewportHeight": 800,
-  "requestTimeout": 10000
+  "projectId": "dk2opw"
```

Comment:
Updated to a project under my personal account for testing. Will need to change back to [the official project ](https://dashboard.cypress.io/projects/fbf96q/runs) once merged.

Please [add a new record key](https://docs.cypress.io/guides/dashboard/projects.html#Set-up-a-project-to-record) to [GitHub Actions secrets](https://help.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets) under the name `CYPRESS_RECORD_KEY`.